fe/fe-core/src/test/java/com/starrocks/analysis/CreateRoutineLoadStmtTest.java

```diff
@@ -71,7 +69,37 @@ public static void beforeClass() throws Exception {
     }
 
     @Test
-    public void testAnalyzeWithDuplicateProperty(@Injectable Analyzer analyzer) throws UserException {
+    public void testParser() {
+        String sql = "CREATE ROUTINE LOAD testdb.routine_name ON table1\n" + "WHERE k1 > 100 and k2 like \"%starrocks%\",\n" + "COLUMNS(k1, k2, k3 = k1 + k2),\n" + "COLUMNS TERMINATED BY \"\\t\",\n" + "PARTITION(p1,p2) \n" + "PROPERTIES\n" + "(\n" + "    \"desired_concurrent_number\"=\"3\",\n" + "    \"max_batch_interval\" = \"20\",\n" + "    \"strict_mode\" = \"false\",\n" + "    \"timezone\" = \"Asia/Shanghai\"\n" + ")\n" + "FROM KAFKA\n" + "(\n" + "    \"kafka_broker_list\" = \"kafkahost1:9092,kafkahost2:9092\",\n" + "    \"kafka_topic\" = \"topictest\"\n" + ");";
```

Comment:
split it to multi lines

Reply:
[split it to multi lines](https://github.com/StarRocks/starrocks/pull/9981/commits/a54ef41417f74a702d7a3543c428aa2b2cf8c655)